src/antares/model/area.py

```diff
@@ -299,6 +299,9 @@ def get_reserves_matrix(self) -> pd.DataFrame:
 
     def get_misc_gen_matrix(self) -> pd.DataFrame:
         return self._area_service.get_misc_gen_matrix(self)
+    
+    def get_renewable_matrix(self, renewable: RenewableCluster) -> pd.DataFrame:
```

Comment:
You should only give a renewable_id not a renewable object

Reply:
Ok i modified the signature, and changed accordingly the signature on renewable_api as well as the unit tests associated